pkgs/by-name/gh/gh-notify/package.nix

```diff
@@ -0,0 +1,34 @@
+{ lib
+, fetchFromGitHub
+, stdenv
+, fzf
+, python3
+}:
+
+stdenv.mkDerivation {
+  pname = "gh-notify";
+  version = "unstable-2024-03-19";
+
+  src = fetchFromGitHub {
+    owner = "meiji163";
+    repo = "gh-notify";
+    rev = "0d8fa377d79cfef0f66d2f03a5921a5e598e6807";
+    hash = "sha256-Ao6gUtgW7enVlWBQhlQDc8ZW/gP90atc2F4rDNUnjj8=";
+  };
+
+  propagatedBuildInputs = [
```

Comment:
The final script should be wrapped in `lib.makeBinPath` these packages, not adding to `propagatedBuildInputs`